src/libraries/Common/tests/System/IO/Compression/CompressionStreamUnitTestBase.cs

```diff
@@ -139,7 +139,7 @@ public virtual async Task Dispose_WithUnfinishedReadAsync()
 
             using (var readStream = await ManualSyncMemoryStream.GetStreamFromFileAsync(compressedPath, false))
             {
-                var decompressor = CreateStream(readStream, CompressionMode.Decompress, true);
+                using var decompressor = CreateStream(readStream, CompressionMode.Decompress, true);
```

Comment:
Personally, I'm not on fan of using braceless usings anywhere (including tests).

Just thought I'd throw it out there.

Reply:
Not true. I got convinced that array-like pattern matching was better than hand writing the equivalent.  And `?.` is pretty good, too :).